### ISSUE TITLE
Feature request: add button to Metadata Viewer to fill metadata for conventional products  

### ISSUE THREAD

**Issue by aloboa** [EnMAP-Box/enmap-box#1052]
Manually filling metadata is tedious and, in most cases, many images share the same metadata.
A button to add conventional metadata for eg the different kinds of Landsat, ASTER, EnMap, Sentinel images 
(or even letting the user provide a csv file) would be very useful.

**Comment by janzandr** [comment]
> A button to add conventional metadata for eg the different kinds of Landsat, ASTER, EnMap, Sentinel images

For conventional sensor product, we have special Import Algos that will take care of proper metadata assignment:
![image](https://github.com/user-attachments/assets/fb6f9388-3602-445b-a25e-ce858432f836)

> or even letting the user provide a csv file

We now support STAC-like sidecar files for providing metadata. Maybe that is what you need (?)
https://enmap-box.readthedocs.io/en/latest/usr_section/usr_manual/metadata_guide.html#

**Comment by janzandr** [comment]
@jakimowb in the GDAL Metadata widget, we have the option to Paste Metadata:
![image](https://github.com/user-attachments/assets/b749765a-49bd-4477-9c40-67a1d6cb3a57)

I'm not really sure how that works. From where would I copy the original metadata? 
What is the format... a table, a json-dict?
Is that feature documented?

**Comment by aloboa** [comment]
> For conventional sensor product, we have special Import Algos that will take care of proper metadata assignment

Yes of course, I know that. But in many cases, the user has an image (eg a Landsat image) that does not come from your Import tool. In those cases, adding the conventional metadata at once would be  useful.

> We now support STAC-like sidecar files for providing metadata. Maybe that is what you need (?)
> https://enmap-box.readthedocs.io/en/latest/usr_section/usr_manual/metadata_guide.html#

What I need is a way to write those files. For example, how do I create a GDAL PAM file out of hdr file?

> @jakimowb in the GDAL Metadata widget, we have the option to Paste Metadata:

I get the Paste option always greyed, even if I select  Copy metadata first...

**Comment by janzandr** [comment]
> Yes of course, I know that. But in many cases, the user has an image (eg a Landsat image) that does not come from your Import tool. In those cases, adding the conventional metadata at once would be useful.

I understand, but I guess that we won't take the time for such convinience function. 
@jakimowb any thought on that?

> What I need is a way to write those files. For example, how do I create a GDAL PAM file out of hdr file?

You can't do this in the GUI.
You have to use GDAL API to do it. Just ask ChatGPT: "How set band metadata with PyGDAL."

> I get the Paste option always greyed, even if I select Copy metadata first...

Hmm, you are right, looks like a bug.
@jakimowb please have a look at this.
